docs/source/en/tasks/masked_language_modeling.md

```diff
@@ -133,7 +133,7 @@ Here is a first preprocessing function to join the list of strings for each exam
 
 ```py
 >>> def preprocess_function(examples):
-...     return tokenizer([" ".join(x) for x in examples["answers.text"]])
+...     return tokenizer(" ".join(eli5["train"][0]["answers"]["text"]))
```

Comment:
This should still use the `examples` passed as input, no? It's the preprocess function that's used below with the `map` of the dataset

Reply:
Yes, sure it has to be examples indeed. I pushed some test code
I think it should be return tokenizer(" ".join(examples["answers"]["text"])), to tokenize all the sentences as one string. If not, i am not understanding it. I thought it would be wrong to tokenize all separate letters of the sentences.
But with this change i am getting "typerror: list indices must be integers of slices, not str", so propabely not a correct pr